pkg/worker/redis.go

```diff
@@ -201,7 +201,10 @@ func (w *RedisWorker) processJob(ctx context.Context, job *Job) {
 	ctx = contextLogger(ctx, job)
 	if h, ok := w.handlers[job.Type]; ok {
 		cCtx, cFunc := context.WithTimeout(ctx, config.Worker.Timeout)
-		defer cFunc()
+		defer func() {
+			zerolog.Ctx(ctx).Error().Str("job_id_timeout", job.ID.String()).Msg("Job cancelled")
```

Comment:
Would this be more meaningful as `job_id: job.ID, timeout: true` ?
Will this include a trace in glitchtip you think, even without `Err()`? 
If not, I'd really love to include also param name of the job, so we also have that info.

Reply:
Done.